backend/templates/partials/head.html

```diff
@@ -86,6 +86,10 @@
 
   <script src="{% static "js/codemirror/modes/sparql/sparql.js" %}"></script>
   <script src="{% static "js/codemirror/modes/sparql/sparql-hint.js" %}"></script>
+  
+  <!-- WASM formatter -->
+  <script src="{% static "js/formatter\index.js" %}"></script>
```

Comment:
Is there a reason for using `/` and `\` here?

Reply:
Does `js/formatter/index.js` load the wasm when being run? The WASM is quite large. It is about as large as the rest of the page on load. Does this block the rendering? Can it be loaded lazily?

Reply:
Nope, no reason for the `/`.
I`m quiet confused that it worked tbh.

Reply:
Concerning the wasm beeing loaded.
That`s a good question.
I know that the module is a "async module", so i don`t thing it will block rendering.
Webpack has a experimental flag [lazyCompilation](https://webpack.js.org/configuration/experiments/#experimentslazycompilation), i did not look into it yet.